package Chapter8;

class UnsupportedFuctionException extends RuntimeException {
	final private int ERR_CODE = 100;
	
	S
	
	public UnsupportedFuctionException(String message, int errCode) {
		super(message);
		// TODO Auto-generated constructor stub
	}

	public int getErrorCode() {return ERR_CODE;}
	
}

class Q8_9
{
	public static void main(String[] args) throws Exception
	{
		throw new UnsupportedFuctionException(" .",100); 지원하지 않는 기능입니다
	}
}